### PR TITLE
Refactor Routing

### PR DESCRIPTION
**Pull request by hmayenve**
## Description

* This PR is to clean up the admin routing from the dashboard

## Relating Ticket
* [SAR-95](https://jira.amida.com/browse/SAR-95)

## How to test

* Run `yarn start`
* You should notice that there is no longer a notus menu on the right hand side of the screen
* You can also notice that routing urls no longer contain `/admin`

### CODE REVIEW

**Review by eawest2 — APPROVED** [review]
Cloned repo, stood it up. Changes reflected as described.

File: src/index.js [hunk 1]
```diff
@@ -15,18 +15,19 @@ import Auth from "layouts/Auth.js";
 import Landing from "views/Landing.js";
 import Profile from "views/Profile.js";
 
+
 ReactDOM.render(
   <BrowserRouter>
     <Switch>
       {/* add routes with layouts */}
-      <Route path="/admin" component={Admin} />
+      <Route path="/" component={Admin} />
```
Comment (jmfadeley) [inline src/index.js:23]: So this is fine for now. I almost wonder if we want a separate admin component, but that's a discussion for another time.

**Review by jmfadeley — APPROVED** [review]
This looks good. May want to discuss routing non-known endpoints to a 404 page. So if go to `http://localhost:3000/whatever`, I go nowhere.

**Comment by hmayenve** [comment]
> This looks good. May want to discuss routing non-known endpoints to a 404 page. So if go to `http://localhost:3000/whatever`, I go nowhere.

I'll make this a ticket ffr

**Pull request merged**